reverie-ptrace/src/vdso.rs

```diff
@@ -87,26 +100,36 @@ mod vdso_syms {
     //    See also
     //    https://developer.arm.com/documentation/ddi0596/2021-06/Base-Instructions/BTI--Branch-Target-Identification-
 
-    pub const clock_getres: &[u8; 16] = &[
+    #[repr(align(64))]
+    struct BufferAligned8([u8;8]);
+    struct BufferAligned16([u8;16]);
```

Comment:
Shouldn't `BufferAligned16` have `#[repr(align(64))]` as well?

Reply:
Hmm, maybe all your changes haven't been pushed, because I'm not seeing the more generic `BufferAligned<size>`.

Reply:
Indeed, sorry :)